src/MetaModels/DcGeneral/Events/Table/Attribute/Subscriber.php

```diff
@@ -519,24 +515,4 @@ public function handleUpdateAttribute(PostPersistModelEvent $event)
             }
         }
     }
-
-    /**
-     * Handle the deletion of an attribute and all attached data.
-     *
-     * @param PreDeleteModelEvent $event The event.
-     *
-     * @return void
-     */
-    public function handleDeleteAttribute(PreDeleteModelEvent $event)
-    {
-        if (($event->getEnvironment()->getDataDefinition()->getName() !== 'tl_metamodel_attribute')) {
-            return;
-        }
-
-        $attribute = $this->createAttributeInstance($event->getModel()->getPropertiesAsArray());
-
-        if ($attribute) {
-            $attribute->destroyAUX();
```

Comment:
The child model is deleted, yes. The problem is however, that the line `$attribute->destroyAUX();` is not executed anymore.
Therefore this method has to remain exactly as it is.